features/rerun_formatter.feature

```diff
@@ -85,3 +85,19 @@ Feature: Rerun Formatter
       | A - ambiguous |
       | B - pending   |
       | C - undefined |
+
+  Scenario: rerun file with trailing new line
+    Given a file named "@rerun.txt" with:
+      """
+      features/c.feature:2
+
+      """
+    When I run cucumber.js with `-f json @rerun.txt`
+    Then the exit status should be 0
+
+  Scenario: empty rerun file
+    Given a file named "@rerun.txt" with:
+      """
+      """
```

Comment:
Please create a new step (or use an existing step if it exists) so this can read something like:
```gherkin
Given an empty file named "@rerun.txt"
```